lib/teaspoon/configuration.rb

```diff
@@ -152,6 +153,12 @@ def self.formatters
       @@formatters.to_s.split(/,\s?/)
     end
 
+    def self.coverage_ignored_files
+      return [] if @@coverage_ignored_files.blank?
```

Comment:
Is it necessary for this line to be here? Since the default is an empty array, we should be safe, no?